conf/gunicorn-dev.conf.py

```diff
@@ -0,0 +1,3 @@
+bind = "0.0.0.0:5000"
+graceful_timeout = 0
+timeout = 0
```

Comment:
I think it's better to have separate `gunicorn*.conf.py` config files for Gunicorn: 

* It's a more obvious place to put the Gunicorn config settings, rather than putting them in a `[server:main]` block in the PasteDeploy config file (you wouldn't think to look for Gunicorn settings there, and it's not obvious that they are Gunicorn settings even once you've found them).
* There are some Gunicorn settings (such as server hooks) that can _only_ go in a Gunicorn config file, not a PasteDeploy one.
* Gunicorn only reads general Gunicorn settings from a PasteDeploy file when run via `pserve`, and we're not using `pserve` anymore